### PR TITLE
tensorflow: 1.9.0 -> 1.11.0

### PR DESCRIPTION
**Pull request by jyp**
###### Motivation for this change

Upstream update

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by globin** [comment]
@GrahamcOfBorg build python3.pkgs.Keras python2.pkgs.Keras

**Comment by globin** [comment]
@GrahamcOfBorg build python3.pkgs.tensorflow python2.pkgs.tensorflow

File: pkgs/development/libraries/protobuf/3.6.nix [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{ callPackage, lib, ... }:
+
+lib.overrideDerivation (callPackage ./generic-v3.nix {
```
Comment (symphorien) [inline pkgs/development/libraries/protobuf/3.6.nix]: you should use overrideAttrs instead according to https://nixos.org/nixpkgs/manual/#sec-pkg-overrideDerivation

**Comment by xeji** [comment]
@GrahamcOfBorg build protobuf3_6 python3.pkgs.tensorflow python2.pkgs.tensorflow python3.pkgs.tensorflow-tensorboard python2.pkgs.tensorflow-tensorboard

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -9863,7 +9863,7 @@ in {
   protobuf = callPackage ../development/python-modules/protobuf {
     disabled = isPyPy;
     doCheck = !isPy3k;
-    protobuf = pkgs.protobuf3_5;
+    protobuf = pkgs.protobuf3_6;
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: This change affects more packages that will going to need to be checked.

**Comment by jyp** [comment]
@xeji note that tensorboard is a dependency of tensorflow (weird I know).
@FRidh right, please let me know if I'm not doing things as they should be done.

**Comment by orivej** [comment]
@jyp Could you rebase onto the current `master`? Since #47353 Protobuf 3.6 is the default.

**Comment by jyp** [comment]
@orivej Done, thanks for the heads up.
@FRidh Please re-consider merging this :)

**Comment by globin** [comment]
Could you remove the old hashes file, otherwise looks fine.

**Comment by jyp** [comment]
@globin I have removed the old hashes. Also, tensorflow 1.11 was released, so I have updated to that.

**Comment by globin** [comment]
@GrahamcOfBorg build python3.pkgs.tensorflow python2.pkgs.tensorflow

**Pull request merged**